cli.js

```diff
@@ -91,6 +98,12 @@ async function init() {
 	ora.text = 'Creating icon';
 	const composedIconPath = await composeIcon(path.join(appPath, 'Contents/Resources', `${appIconName}.icns`));
 
+	ora.text = 'Checking minimum runtime';
+	const {stdout: minSystemVersion} = await execa('/usr/libexec/PlistBuddy', ['-c', 'Print :LSMinimumSystemVersion', infoPlistPath]);
+	const minorVersion = Number(minSystemVersion.replace('10.', '')) || 0;
```

Comment:
This check is not good enough:

```js
'10.10.2'.replace('10.', '')
//=> "10.2"
```

Reply:
Good catch. Added some modified logic, hope you agree. It's not as sophisticated as a true 'version compare' but hopefully will be sufficient. Otherwise, we can replace this with a true version compare logic (which can compare 10.10.1a for example).

Reply:
On our usage we've noticed one machine was making bad DMGs.
I'm now investigating the Xcode diffs. but this logic will fail if no `LSMinimumSystemVersion` exists.

Reply:
@talaviram normally this is included by default in XCode. Are you running into a situation where it's not?

Something like:
> otool -l /usr/local/go/bin/go | grep -B1 -A3 MIN_MACOS

Can also be used to define min versioning, but it will need some more work to incorporate.